core/v0/api/core.yaml

```diff
@@ -1563,7 +1564,7 @@ components:
         tags:
           $ref: '#/components/schemas/Tags'
     Context:
-      description: Describes a beckn message context
+      description: Describes the context of the message being sent.<br>This has properties like domain,country,city,action,code_version,bap_id,bap_uri,bpp_id,bpp_uri,transaction_id,message-id,timestamp,key,ttl
```

Comment:
`"Describes the context of the message"` may be changed to `"Describes the context header of the message"`